typings/internalBinding.d.ts

```diff
@@ -439,3 +439,262 @@ declare function InternalBinding(binding: 'config'): {
   bits: number,
   hasDtrace: boolean
 }
+
+declare namespace InternalFSBinding {
+  class FSReqCallback<ResultType = unknown> {
+    constructor(bigint?: boolean);
+    oncomplete: ((error: Error) => void) | ((error: null, result: ResultType) => void);
+    context: any;
+  }
+
+  interface FSSyncContext {
+    fd?: number;
+    path?: string;
+    dest?: string;
+    errno?: string;
+    message?: string;
+    syscall?: string;
+    error?: Error;
+  }
+
+  type Buffer = Uint8Array;
+  type StringOrBuffer = string | Buffer;
+
+  const kUsePromises: symbol;
```

Comment:
I wanted to make this a `unique symbol` but it doesn't work.
`const { kUsePromises } = internalBinding('fs')` is still considered a `symbol` and that is then incompatible with the function signatures.

Reply:
this sounds like something to notify typescript about in an issue

Reply:
Opened https://github.com/microsoft/TypeScript/issues/43657